### DISCUSSION TITLE
There is nothing to stdin/stderr?

### DISCUSSION THREAD

**Discussion by jayhohoho2019** [Chia-Network/bladebit#62] [General]
I compiled it on Ubuntu 20.04 LTS and started the first test run but there is nothing written to stdin.  Not sure what's happening, although the # of threads I've specified are showing 100% usage, and RAM usage is ramping up towards 400G when I write this.  There is no logging?

**Comment by jayhohoho2019** [comment]
It turns out this was caused by my trying to pipe to "tee" like I was doing with the madmax plotter.

**Comment by mocksoul** [comment]
see #22 with fix for that

**Reply by jayhohoho2019** [reply]
Thanks.  Applied.  Could this be merged?
